src/client/job.rs

```diff
@@ -76,7 +112,7 @@ fn job_status_with_counts_cells(info: &JobInfo) -> Vec<CellStruct> {
             );
         }
     };
-    let mut result: Vec<CellStruct> = Vec::new();
+    let mut result: Vec<CellStruct> = vec![job_progress_bar(info).cell()];
```

Comment:
Maybe disable progress when it is not a task array? (I do not know, just thinking)

Reply:
I would keep it also for jobs with a single task, for consistency and fixed table layout (mainly for tests).